kaldb/src/main/java/com/slack/kaldb/recovery/RecoveryService.java

```diff
@@ -281,9 +289,25 @@ boolean handleRecoveryTask(RecoveryTaskMetadata recoveryTaskMetadata) {
       } catch (Exception ex) {
         LOG.error("Exception in recovery task [{}]: {}", validatedRecoveryTask, ex);
         return false;
+      } finally {
+        long endTime = System.nanoTime();
+        LOG.info(
+            "Recovery task {} took {}µs, (subtask times {}, {}, {}, {})",
```

Comment:
I think it would be great to prefix the sub-task time so we can understand the time without looking at the code. 

Also, may be time we look at using a structured logging library.

Reply:
https://github.com/slackhq/kaldb/pull/441/commits/d19a91277361f5fe82ddfeaa44b48afff1e892ad